src/domains/charts/providers/menu/menu.js

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+import styled from "styled-components"
+import { Flex } from "@netdata/netdata-ui"
+import { MenuGroupContainer } from "domains/charts/providers/menuGroup"
+import { ActiveMenuConsumer } from "domains/charts/providers/active"
+import { MenuConsumer } from "./context"
+
+export const Menu = ({ ids, activeMenuId, onMenuItemClick, onGroupItemClick, ...rest }) => (
+  <Flex as="ul" column {...rest}>
```

Comment:
if its a `ul` then children should be `li`

Reply:
Yes, it is :)